piglow/examples/strobe.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"time"
+
+	"github.com/goiot/devices/piglow"
+	"golang.org/x/exp/io/i2c"
+)
+
+func main() {
+	p, err := piglow.Open(&i2c.Devfs{Dev: "/dev/i2c-1", Addr: 0x54})
+	if err != nil {
+		panic(err)
+	}
+
+	defer p.Close()
+
+	if err := p.Reset(); err != nil {
+		panic(err)
+	}
+
+	for {
```

Comment:
There is no need to do that. When processes finishes, its resources will be returned back. Signal handling just complicates the code. We are just defer closing because this is a reference code and can be copy pasted to a non main function.